packages/terra-modal-manager/package.json

```diff
@@ -37,7 +37,7 @@
   },
   "devDependencies": {
     "terra-collapsible-menu-view": "^6.35.0",
-    "terra-form-select": "^5.0.0"
+    "terra-form-select": "^6.8.0"
```

Comment:
Where is this dependency being used?

Reply:
It is used in the test [DisclosureIntegrationComponent.jsx](https://github.com/cerner/terra-framework/blob/master/packages/terra-modal-manager/src/terra-dev-site/test/modal-manager/DisclosureIntegrationComponent.jsx#L4)

Reply:
Figured it was but missed it when I checked